backend/typescript/models/camper.model.ts

```diff
@@ -9,7 +9,7 @@ export interface Camper extends Document {
   parentName: string;
   contactEmail: string;
   contactNumber: string;
-  camps: Schema.Types.ObjectId;
+  camps: Schema.Types.ObjectId[];
```

Comment:
can you merge in the latest changes? this field should have been changed in udy's pr